public/app/features/alerting/unified/NotificationPolicies.test.tsx

```diff
@@ -636,102 +385,39 @@ describe('NotificationPolicies', () => {
       },
     });
 
-    jest.spyOn(alertmanagerApi, 'useGetAlertmanagerAlertGroupsQuery').mockImplementation(() => ({
-      currentData: [],
-      refetch: jest.fn(),
-    }));
-
     renderNotificationPolicies(dataSources.promAlertManager.name);
-    const rootRouteContainer = await ui.rootRouteContainer.find();
-    await waitFor(() =>
-      expect(within(rootRouteContainer).getByTestId('matching-instances')).toHaveTextContent('0instance')
-    );
 
-    expect(ui.editButton.query(rootRouteContainer)).not.toBeInTheDocument();
-    expect(ui.newPolicyCTAButton.query()).not.toBeInTheDocument();
-    expect(mocks.api.fetchAlertManagerConfig).not.toHaveBeenCalled();
-    expect(mocks.api.fetchStatus).toHaveBeenCalledTimes(1);
-  });
+    expect(await ui.rootRouteContainer.find()).toBeInTheDocument();
 
-  it.skip('Can add a mute timing to a route', async () => {
-    const defaultConfig: AlertManagerCortexConfig = {
-      alertmanager_config: {
-        receivers: [{ name: 'default' }, { name: 'critical' }],
-        route: {
-          continue: false,
-          receiver: 'default',
-          group_by: ['alertname'],
-          routes: [simpleRoute],
-          group_interval: '4m',
-          group_wait: '1m',
-          repeat_interval: '5h',
-        },
-        templates: [],
-        mute_time_intervals: [muteInterval],
-      },
-      template_files: {},
-    };
-
-    const currentConfig = { current: defaultConfig };
-    mocks.api.updateAlertManagerConfig.mockImplementation((amSourceName, newConfig) => {
-      currentConfig.current = newConfig;
-      return Promise.resolve();
-    });
+    expect(ui.newChildPolicyButton.query()).not.toBeInTheDocument();
+    expect(ui.newSiblingPolicyButton.query()).not.toBeInTheDocument();
+  });
 
-    mocks.api.fetchAlertManagerConfig.mockResolvedValue(defaultConfig);
+  it('Can add a mute timing to a route', async () => {
```

Comment:
(We need to revamp some of the tests in general once the alertingApiServer is always turned on. We should make the k8s APIs all drive from the corresponding `grafana` AM mock config that we have/remove that config altogether)